app/routes/wow.ilvl-shopping-list.tsx

```
@@ -0,0 +1,368 @@
+          onChange={(e) => setMaxPurchasePrice(e.currentTarget.value)}
+        />
+        <InputWithLabel
+          labelTitle="Minimum Item Level"
+          name="desiredMinIlvl"
+          type="number"
+          value={desiredMinIlvl}
+          min={0}
+          onChange={(e) => setDesiredMinIlvl(e.currentTarget.value)}
+        />
+        <div className="flex flex-col gap-2">
+          <label className="text-sm font-medium">Desired Stats</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)